app/com/gu/contentapi/sanity/ShowBlocksTest.scala

```diff
@@ -0,0 +1,73 @@
+package com.gu.contentapi.sanity
+
+import com.gu.contentapi.sanity.support.TestFailureHandler
+import play.api.libs.json.{JsArray, JsObject, JsValue, Json}
+
+class ShowBlocksTest(testFailureHandler: TestFailureHandler) extends SanityTestBase(testFailureHandler) {
+
+  "The newest Flex items in the search results" should "include blocks if show-blocks=all" in {
+
+    val httpRequest = requestHost("search?show-blocks=all&show-fields=internalComposerCode").get()
+    whenReady(httpRequest) { result =>
+      assume(result.status == 200, "Service is down")
+      val json = Json.parse(result.body)
+      val newestItemList = (json \ "response" \ "results").asOpt[List[Map[String, JsValue]]]
+      for (item <- newestItemList.value) {
+        if ((item("fields") \ "internalComposerCode").asOpt[String].isDefined) {
+          checkBlocks(item)
+        }
+      }
+    }
+
+  }
+
+  "The newest Flex item" should "include blocks if show-blocks=all" in {
+
+    // First request to get the latest items
+    val httpRequest = requestHost("search?show-fields=internalComposerCode").get()
+    whenReady(httpRequest) { result =>
+      assume(result.status == 200, "Service is down")
+      val json = Json.parse(result.body)
+      val newestItemList = (json \ "response" \ "results").as[List[Map[String, JsValue]]]
+      val newestFlexItem = newestItemList.find(item => (item("fields") \ "internalComposerCode").asOpt[String].isDefined)
+      newestFlexItem foreach { item =>
+        // Second request to the item endpoint for the first flexible content item in the search results
+        val itemUrl = item("id").as[String] + "?show-blocks=all"
```

Comment:
I think this would be better named `itemPath` or some such as it does not contain the full URL?